src/expire.c

```diff
@@ -747,6 +747,6 @@ void persistCommand(client *c) {
 void touchCommand(client *c) {
     int touched = 0;
     for (int j = 1; j < c->argc; j++)
-        if (lookupKeyRead(c->db,c->argv[j]) != NULL) touched++;
+        if (lookupKeyReadWithFlags(c->db,c->argv[j],LOOKUP_NONE) != NULL) touched++;
```

Comment:
```suggestion
        int flags = c->flags & CLIENT_STEALTHMODE ? LOOKUP_NOTOUCH : LOOKUP_NONE;
        if (lookupKeyReadWithFlags(c->db,c->argv[j],flags) != NULL) touched++;
```

Reply:
I deliberately preserved `touchCommand`'s function. If we only want LFU/LRU stats left unchanged, we can just avoid calling this command. It is unnecessary to disable this command in stealth mode.